src/storage-controller/src/history.rs

```diff
@@ -204,7 +204,7 @@ impl<T: timely::progress::Timestamp + TotalOrder> CommandHistory<T> {
         }
 
         let count = u64::cast_from(run_sinks.len());
-        self.metrics.run_ingestions_count.set(count);
```

Comment:
🙈

Reply:
I believe this one is actually my fault because I added the storage command history, including metrics 🫠